dartsim/src/JointFeatures_TEST.cc

```diff
@@ -163,6 +166,473 @@ TEST_F(JointFeaturesFixture, JointSetCommand)
   }
 }
 
+TEST_F(JointFeaturesFixture, JointSetPositionLimitsWithForceControl)
```

Comment:
Also because we need to support DART 6.9, please wrap these tests in 
```c++
// From upstream 6.10.0 or OSRF 6.10.0~osrf19~2021-06-10
#if DART_VERSION_AT_LEAST(6, 10, 0)
...
#endif
```

Reply:
https://github.com/ignitionrobotics/ign-physics/pull/260/commits/4984ceca07eac786607ec3275066b28a17271afd